client/rpc-servers/src/lib.rs

```diff
@@ -27,8 +27,10 @@ use jsonrpc_core::{IoHandlerExtension, MetaIoHandler};
 use log::error;
 use pubsub::PubSubMetadata;
 
+const MEGABYTE: usize = 1024 * 1024;
+
 /// Maximal payload accepted by RPC servers.
-pub const MAX_PAYLOAD: usize = 15 * 1024 * 1024;
+pub const DEFAULT_MAX_PAYLOAD: usize = 15 * MEGABYTE;
```

Comment:
```suggestion
pub const RPC_MAX_PAYLOAD_DEFAULT: usize = 15 * MEGABYTE;
```
…there seem to be a pattern to use `XXX_` as a prefix.